Dockerfile

```diff
@@ -5,7 +5,7 @@ WORKDIR ${HOME}
 
 # install Chrome
 RUN curl -sL http://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
-RUN echo 'deb http://dl.google.com/linux/chrome/deb/ stable main' >> /etc/apt/sources.list.d/google.list
+RUN echo 'deb [trusted=yes] http://dl.google.com/linux/chrome/deb/ stable main' >> /etc/apt/sources.list.d/google.list
```

Comment:
Do we remove this?

Reply:
Only if it started working again (i.e. Google updated its expired key)

Reply:
This is outdated - we checked this morning and its working again :+1: